modules/database/database.py

```
@@ -0,0 +1,151 @@
+        :param library_name: The name of the library
+        :type library_name: str
+        :param webhook_id: The ID of the webhook
+        :type webhook_id: int
+        :return: The recently added item that was added
+        """
+        try:
+            return self._create_entry(table_schema=RecentlyAddedItem,
+                                      name=name,
+                                      library_name=library_name,
+                                      webhook_id=webhook_id) # type: ignore
+        except Exception as e:
```

Ruff (0.8.0)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)